public/stylesheets/styles.css

```diff
@@ -1,22 +1,21 @@
 :root {
   --green: #008000;
   --grey: #333333;
+  --dark-green: #0a6c0a;
+  --dark-blue: #0b47c7;
 }
 
 * {
   font-family: Arial, Helvetica, sans-serif;
 }
 
 body {
-  padding: 0 0.5em;
   margin-bottom: 5em;
 }
 
 h1,
-h2,
-h3,
-h4 {
-  color: var(--green);
+h2 {
+  color: var(--dark-green);
```

Comment:
Looks like you deleted h3 and h4. Can you leave them there? I use them for one of my pages

Reply:
yes, I must have accidentally remoted when I solve for the conflict sorry!